decidim-meetings/app/assets/javascripts/decidim/meetings/meetings_filters.js.es6

```diff
@@ -33,14 +33,19 @@ $(document).ready(() => {
     }
 
     let scopeValues = location.match(/scope_id\[\]=([^&]*)/g);
-
     if (scopeValues) {
       scopeValues = scopeValues.map(val => val.match(/scope_id\[\]=(.*)/)[1]);
       scopeValues.forEach(value => {
         $form.find(`input[type=checkbox][value=${value}]`)[0].checked = true;
       })
     }
 
+    let categoryIdValues = location.match(/filter\[category_id\]=([^&]*)/g);
```

Comment:
I think it will be more maintainable if we move those regex into separate variables or just factor out some kind of function to extract the params. We can review it later @mrcasals